src/services/api.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'https://talk-and-travel.pp.ua/api/authentication';
+
+axios.interceptors.request.use(request => {
+  console.log('Starting Request', request);
```

Comment:
delete it

Reply:
Should I delete iterseptors?

Reply:
Those are do nothing actually right now so let's delete them.